src/software/ai/hl/stp/play/example_play_test.cpp

```diff
@@ -1,41 +0,0 @@
-#include "software/ai/hl/stp/play/example_play.h"
```

Comment:
This might be elsewhere, but showing how plays will be tested seems like a pretty key point (might just be sim tests, which seems like a reasonable approach to me).

Reply:
These are the only non-sim play tests. I'm thinking about moving simplaytests to the play folder